src/main/java/com/google/sps/server/ScheduleCalendar.java

```diff
@@ -115,46 +107,33 @@ public String getCalendarId() {
    * Helper function to add a course in the schedule to the calendar, as an event. 
    *
    * @param course the course being added 
-   * @param startTime the start time for the calendar event
-   * @param endTime the end time for the calendar event
+   * @param startTime the start time of the course
+   * @param endTime the end time of the course
+   * @param termLength the number of weeks in the user's college term
    */
-  private void addEvent(ScheduledCourse course, String startTime, String endTime) throws IOException {
+  private void addEvent(Course course, ZonedDateTime startTime, ZonedDateTime endTime, int termLength) throws IOException {
```

Comment:
If Course is only being used for it's name, it might be worth just passing in the name of the course as a string (feel free to not make this change if you plan on using more course info down the line)

Reply:
I was thinking of expanding the fields for calendar events, so I'm not going to change it right now, but good catch. Thanks!